src/CddInterface.c

```diff
@@ -38,8 +27,8 @@ extern void dd_SetLinearity(dd_MatrixPtr, char *);
 // }
 
 // The following conversion has been taken from
-// https://github.com/gap-packages/NormalizInterface/src
-// Thanks to Sebastian Gutsche
+// https://github.com/gap-packages/NormalizInterface
+// Thanks to Max Horn
```

Comment:
Just to be clear, I am also fine with not getting thanked here, but if there are thanks, then to the author, I'd say ;-).